### ISSUE TITLE
Question about the Bitshift between attn@V

### ISSUE THREAD

**Issue by csguoh** [megvii-research/FQ-ViT#49]
hi, authors
In eq.16, I noticed you have used the Attn@Vq = 2**(-Attn_q) @ V_q = V_q >> Attn_q. However, to my knowledge, the second equation is not hold, i.e.,   2**(-Attn_q) @ V_q != V_q >> Attn_q, since the orignal attention is matmul, however the bitshift works in element-wise product. I wonder if the eq16 confuse these two mat operator?

**Comment by shawnnjupt** [comment]
That is true, eq16 is not right in matmul. I think the author makes some mistakes in eq16 ,but i can get their idea
